tests/test_translate.py

```diff
@@ -17,6 +17,13 @@ def test_translate_process_graph(self):
         print(graph)
         assert True
 
+    def test_translate_process_graph_none(self):
```

Comment:
How about naming this to `test_translate_process_graph_no_params` or `test_translate_process_graph_none_params`? A process graph is given, but no parameters.

Reply:
Sure I can change this